pkgs/applications/networking/cluster/starboard/default.nix

```
@@ -0,0 +1,40 @@
+{ lib, buildGoModule, fetchFromGitHub }:
+
+buildGoModule rec {
+  pname = "starboard";
+  version = "0.4.0";
+
+  src = fetchFromGitHub {
+    owner = "aquasecurity";
+    repo = pname;
+    rev = "v${version}";
+    sha256 = "0jvr8lf3wdg6svsfi3fdknqw543495gysznbqhl42w7d3r5krcyw";
+  };
+
+  vendorSha256 = "0p5svprhbnb7mfln1mhaq55w8xnj8v3sinwkysxjzh1g8p36mglp";
+
+  doCheck = false;
+
+  buildFlagsArray = [
+    "-ldflags="
+    "-s"
+    "-w"
+    "-X main.version=v${version}"
+  ];
+
+  meta = with lib; {
+    description = "Kubernetes-native security tool kit.";
```

```suggestion
    description = "Kubernetes-native security tool kit";
```

The `meta.description` has no punctuation marks in the end.

---

Resolved this same issue for trivy pr